release/tag_repos.sh

```diff
@@ -76,6 +76,7 @@ repos=(
 	"runtime"
 	"shim"
 	"tests"
+	"documentation"
```

Comment:
Oh - extra points for adding this in its alphabetical location ;)

Reply:
@jodh-intel Fixed.